includes/options/class-amp-options-menu.php

```diff
@@ -670,6 +681,81 @@ public function render_stories_export() {
 		<?php
 	}
 
+	/**
+	 * Render the stories settings section.
+	 *
+	 * @since 1.3
+	 */
+	public function render_stories_settings() {
+		$this->render_stories_settings_page_advance();
+	}
+
+	/**
+	 * Render the page advancement settings section.
+	 *
+	 * @since 1.3
+	 */
+	private function render_stories_settings_page_advance() {
+		$definitions                = AMP_Story_Post_Type::get_stories_settings_definitions();
+		$auto_advance_after_options = $definitions['auto_advance_after']['data']['options'];
+
+		$story_settings            = AMP_Options_Manager::get_option( 'story_settings' );
+		$story_settings_field_name = sprintf( '%s[%s]', AMP_Options_Manager::OPTION_NAME, AMP_Story_Post_Type::STORY_SETTINGS_OPTION );
+		?>
+		<fieldset>
+			<p>
+				<label for="stories_settings_auto_advance_after">
+					<strong><?php echo esc_html__( 'Advance to next page', 'amp' ); ?></strong>
```

Comment:
In the story editor, there is the "These settings are applied to new pages." help text.

Here, I'd expect a similar help text: "These settings are applied to new stories only."